src/firefly/js/visualize/iv/TileDrawHelper.jsx

```diff
@@ -39,16 +46,22 @@ export function makeImageFromTile(src, pt, width, height, scale,opacity=1) {
 
 
 export function createImageUrl(plot, tile) {
-    const params = {
-        file: tile.url,
-        state: plot.plotState.toJson(),
-        type: 'tile',
-        x: tile.xoff,
-        y: tile.yoff,
-        width: tile.width,
-        height: tile.height
-    };
-    return encodeServerUrl(getRootURL() + 'sticky/FireFly_ImageDownload', params);
+    if (plot.type==='image') {
+        const params = {
```

Comment:
Just curious, do we have only two plot types: 'image' and 'hips'? To make sure the new code does not break anything, we might want to check for 'hips' rather than 'image'... It's not only here, I saw many checks like this.

Reply:
Your right.  I need to make a pass though the code and reevaluate when I use `'image'` and `'hips'`